utils/python/nexus_nei2016_linker.py

```diff
@@ -200,12 +207,15 @@ def get_hemco_simulation_time(file_path):
     parser.add_argument('-w', '--work_dir', help='work directory in the workflow', required=False)
```

Comment:
```suggestion
    parser.add_argument('-w', '--work_dir', help='work directory in the workflow', required=True)
```
Looks to be required